subworkflows/nf-scil/registration/main.nf

```diff
@@ -31,6 +45,31 @@ workflow REGISTRATION {
             image_warped = REGISTRATION_ANATTODWI.out.t1_warped
             transfo_image = REGISTRATION_ANATTODWI.out.transfo_image
             transfo_trk = REGISTRATION_ANATTODWI.out.transfo_trk
+        }
+        else if ( params.run_surgery ) {
+            // ** Set up input channel ** //
+            ch_register = ch_ref.combine(ch_image, by: 0)
+                                .combine(ch_ref_segmentation, by: 0)
+                                .combine(ch_segmentation, by: 0)
+
+            // ** Registration using Easyreg ** //
+            REGISTRATION_EASYREG ( ch_register )
+            ch_versions = ch_versions.mix(REGISTRATION_EASYREG.out.versions.first())
+
+            // ** Setting outputs ** //
+            image_warped = REGISTRATION_EASYREG.out.flo_reg
+            transfo_image = REGISTRATION_EASYREG.out.fwd_field
+                .map{ it + [[]] }
+            transfo_trk = REGISTRATION_EASYREG.out.bak_field
+                .map{ [[]] + it }
+            ref_warped - REGISTRATION_EASYREG.out.ref_reg
```

Comment:
Not sure what this line is doing.

Reply:
Are you referring to ".map{ [[]] + it } or ref_warped output declaration?

Reply:
In fact it is not a = but a -. So I think you have a bug here.